config/routes.rb

```diff
@@ -1,4 +1,10 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+
+  get 'quotes', to: 'quotes#index'
+  # Where we suppose to only have '/quotes' in the url or is the param query string okay?
```

Comment:
Having the query through the params is totally fine! If you didn't want it to come through the params (maybe it's sensitive data you don't want the user to see in the params for whatever reason). You could pass the param through the request body instead. Take a look at how to do that in rails docs.